### PR TITLE
added module games, updated readme, updated help module

### PR DESCRIPTION
**Pull request by jd-apprentice**
- added game modules, for now its only roll_dice
- updated the readme with the new commands
- updated the help module with the new features

### CODE REVIEW

**Comment by jd-apprentice** [comment]
giphy

File: src/modules/games.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+class Games(commands.Cog):
+
+    def __init__(self, bot):
+        self.bot = bot
+    
+    @commands.command()
+    async def roll_dice(self, ctx):
+        e = EmbedGenerator(ctx)
+        e.title = "[DICE] Tira un dado"
+        e.description = "Proba tu suerte tirando un dado!"
```
Comment (JaviCeRodriguez) [inline src/modules/games.py:17]: `¡Probá tu suerte tirando un dado!`

File: src/modules/games.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        e.fields = [
+            ("Tu numero es: ", random.randint(1, 6))
```
Comment (JaviCeRodriguez) [inline src/modules/games.py:19]: `Tu número es`

File: src/modules/help.py [hunk 3]
```diff
@@ -29,4 +29,14 @@ async def welcome(self, ctx):
...
         embed = e.generate_embed()
         
+        await ctx.send(embed=embed)
+    
+    @help.command()
+    async def redes(self, ctx):
+        e = ClassicEmbed(ctx)
+        e.title = "[HELP] Redes"
+        e.description = """Módulo de redes sociales. 
+        El comando es prefix + redes, muestra los links de las redes sociales de Incubator."""
```
Comment (JaviCeRodriguez) [inline src/modules/help.py:39]: Aprovechemos el prefijo:
```py
e.description = f"""Módulo de redes sociales. 
El comando es {self.PREFIX}redes. Muestra los links de las redes sociales de Incubator."""
```

File: src/modules/redes.py [hunk 4]
```diff
@@ -15,7 +15,7 @@ async def redes(self, ctx):
         e.title = "[SOCIALS] Links de redes sociales"
         e.description = "Links de redes sociales de Incubot"
         e.fields = [
-            ("Twitter", "Todavia no disponible"),
+            ("Twitter", "Todavía no disponible"),
             ("Discord", "Es privado panflin"),
             ("Pagina Web", "https://incubator.com.ar/"),
```
Comment (JaviCeRodriguez) [inline src/modules/redes.py:20]: `Página Web`

File: src/modules/redes.py [hunk 5]
```diff
@@ -15,7 +15,7 @@ async def redes(self, ctx):
         e.title = "[SOCIALS] Links de redes sociales"
         e.description = "Links de redes sociales de Incubot"
```
Comment (JaviCeRodriguez) [inline src/modules/redes.py:16]: de Incubator

**Review by JaviCeRodriguez — COMMENTED** [review]
Hay cositas por corregir!

File: src/modules/games.py (hunk 1 above)
Comment (jd-apprentice) [reply src/modules/games.py:17]: Listo

**Review by JaviCeRodriguez — APPROVED** [review]
Casi LGTM

**Pull request merged**